packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -141,15 +141,19 @@ export async function prefetchLatestSnapshot(
 					});
 					assert(cacheP !== undefined, 0x1e7 /* "caching was not performed!" */);
 					await cacheP;
+					// Schedule it to remove from cache after 5s, so that if this snapshot is going to be used on load,
+					// and we don't find it in this cache, then we will make a network call again to fetch this
```

Comment:
It might be hard to follow (for someone not knowing details of snapshot fetching flow).
Maybe
```
Schedule it to remove from cache after 5s.
1. While it's in snapshotNonPersistentCache: Load flow will use this value and will not attempt to access persistent cache or fetch snapshot from network. That's the best from perf POV, but cache is not being updated, so it gets stale after a while, thus we want to eventually remove snapshot from this cache.
2. After it's removed from snapshotNonPersistentCache: snapshot is present in persistent cache, so we sill still use it (in accordance with cache policy controlled by host). But load flow will also fetch snapshot (in parallel) from storage and update cache. This is fine long term, but is an extra cost (unneeded network call).
```